### PR TITLE
Adding appName to webWallet.logout()

### PR DESCRIPTION
**Pull request by dafuga**

### CODE REVIEW

File: src/kit.ts [hunk 1]
```diff
@@ -53,6 +53,11 @@ export interface LoginResult {
     session: Session
 }
 
+export interface LogoutContext {
+    session: SerializedSession | Session
```
Comment (apporc) [inline src/kit.ts]: Can we use only `Session` type here, protocol-scatter is using the `chain` of a session, it should be a `ChainDefinition` type, not a `chainId`, if i am not wrong, then `chain` in `SerializedSession` is only a `chainId`.
https://github.com/wharfkit/protocol-scatter/blob/master/src/index.ts#L24

Comment (dafuga) [reply src/kit.ts]: Done! Let me know if you run into any other issues!

**Review by apporc — APPROVED** [review]

File: src/wallet.ts [hunk 2]
```diff
@@ -135,7 +135,7 @@ export interface WalletPlugin {
      * @returns A promise that resolves when the wallet plugin logout process is complete.
      * @throws An error if the logout could not happen.
      */
-    logout?(session: Session | SerializedSession): Promise<void>
+    logout?(LogoutContext: LogoutContext): Promise<void>
```
Comment (apporc) [inline src/wallet.ts]: `context: LogoutContext`

**Review by apporc — CHANGES_REQUESTED** [review]

File: src/kit.ts [hunk 3]
```diff
@@ -476,6 +481,31 @@ export class SessionKit {
         }
     }
 
+    logoutParams(walletPlugin: WalletPlugin, session?: Session | SerializedSession): LogoutContext {
```
Comment (apporc) [inline src/kit.ts]: session doesn't need to be optional.

File: src/kit.ts [hunk 4]
```diff
@@ -476,6 +481,31 @@ export class SessionKit {
...
+                    chain: this.getChainDefinition(session.chain),
+                    permissionLevel: PermissionLevel.from({
+                        actor: session.actor,
+                        permission: session.permission,
+                    }),
+                    walletPlugin,
+                }),
+                appName: this.appName,
+            }
+        } else {
```
Comment (apporc) [inline src/kit.ts]: this `else` is unnecessary if `session` is a required option.

**Review by apporc — CHANGES_REQUESTED** [review]

File: src/kit.ts (hunk 3 above)
Comment (dafuga) [reply src/kit.ts]: true 😅

**Review by apporc — APPROVED** [review]

**Pull request merged**